admin_notifications/schema.py

```diff
@@ -0,0 +1,67 @@
+import graphene
+from graphene_sqlalchemy import SQLAlchemyObjectType
+from graphql import GraphQLError
+from helpers.auth.authentication import Auth
+from datetime import datetime
+from admin_notifications.models import AdminNotification as \
+    AdminNotificationModel
+
+
+class AdminNotifications(SQLAlchemyObjectType):
+    """
+        Returns the admin_notificationspayload
+    """
+    class Meta:
+        model = AdminNotificationModel
+
+
+class NotificationsList(graphene.ObjectType):
+    """
+        Class to return the types for Admin notifications
+        \n- rooms: The rooms data
+    """
+    notifications = graphene.List(AdminNotifications)
+
+
+class Query(graphene.ObjectType):
+    all_unread_notifications = graphene.Field(
+        NotificationsList,
+        description="Returns a list of admin notifications"
+    )
+
+    @Auth.user_roles('Admin')
+    def resolve_all_unread_notifications(self, info):
```

Comment:
@codinger41, I see that the admin can only view unread notifications from the implementation here. Would it not be a good idea to enable the admin user to view all the notifications, regardless of the read status?
Also, please add the `Super_Admin` role as the super admin should also be granted these privileges.

Reply:
@issaian I don't think the admins need to see notifications that have already been read. I'd add the super admin role.

Reply:
Why do you think admins need to see already read notifications?

Reply:
@codinger41 the implementation without the viewing of read notifications works fine. I thought that there was a mutation to enable the admin to delete notifications. As there is none, there is no need of it then.